src/Auth/CacheService.js

```diff
@@ -0,0 +1,29 @@
+class CacheService {
+    static cachedData = {};
+
+    constructor() {
+        this.invalidCacheWindowInSeconds = 30;
```

Comment:
you use this only in the get() method. no need to declare a variable in this case, just use the value in that method